app/models/rules_factory.rb

```diff
@@ -4,10 +4,12 @@ def initialize(auction)
   end
 
   def create
+    eligibility = EligibilityFactory.new(auction).create
```

Comment:
We can look at this later, but I am a little leery of one Factory encapsulating another, especially since the Eligibility is really only used in `user_can_bid?` (a method I would argue should just be pulled from Rules, although there is the issue of sealed-bid auctions...) But I suppose we can fix that later.